engine/src/main/java/org/hibernate/search/engine/mapper/schema/management/SchemaExport.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.engine.mapper.schema.management;
```

Comment:
Just noticed the package... IMO this would fit better in `engine.common.schema.management`, since it's not strictly about the mapper. Same for `SchemaExportExtension`.